_posts/en/newsletters/2020-04-29-newsletter.md

```diff
@@ -0,0 +1,330 @@
+  the challenges of statically analyzing a Bitcoin Script program in
+  contrast with the Simplicity language.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 0.20.0rc1][bitcoin core 0.20.0] is a release candidate
+  for the next major version of Bitcoin Core.
+
+- [LND 0.10.0-beta.rc5][lnd 0.10.0-beta] allows testing the next major
+  version of LND.
+
+- [C-Lightning 0.8.2-rc2][c-lightning 0.8.2] is the newest release
+  candidate for the next version of C-Lightning.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Rust-Lightning][rust-lightning repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+*Note: the commits to Bitcoin Core mentioned below apply to its master
+development branch and so those changes will likely not be released
+until version 0.21, about six months after the release of the upcoming
+version 0.20.*
+
+- [Bitcoin Core #15761][] adds an `upgradewallet` RPC, doing away
+  with the old method of upgrading upon startup, thereby
+  allowing users to unlock and upgrade their wallets to
+  [Hierarchical Deterministic (HD)][Hierarchical Deterministic ref]
+  while it is loaded.  This addition is also compatible with
+  [multi-wallet][multi-wallet] as it works on the individual wallet that
+  is specified by the RPC.
+
+- [Bitcoin Core #17509][] allows the wallet GUI to save a [Partially
+  Signed Bitcoin Transactions][topic psbt] (PSBT) to a file, as well as
+  load a PSBT from a file.  Saving is available in private-key-disabled
+  wallets where Bitcoin Core previously automatically copied a PSBT to
+  the clipboard (see the PRs described in Newsletters [#74][news74 core
+  psbt] and [#82][news82 core psbt]).  Loading a PSBT will offer to
+  finalize and broadcast the transaction if all signatures are
+  available; otherwise, the PSBT will be copied to the clipboard for
+  signing by a separate user action (for example using an RPC from the
+  GUI console or using a separate tool such as [HWI][topic hwi]).  A
+  follow-up PRs is expected to add the ability to sign PSBTs in the GUI.
+
+## Special thanks
+
+We thank Antoine Riard, ZmnSCPxj, and Matt Corallo for reviewing drafts of this
+newsletter and helping us understand the details of the LN atomicity
+issue.  Any remaining errors are the fault of the newsletter author.
+
+## Footnotes
+[^package-relay]:
+    The ultimate solution to deal with arbitrary feerates in LN also
+    depends on Bitcoin full nodes being able to perform [package
+    relay][Bitcoin Core #14895], a feature that's long been discussed
+    but never satisfactorily implemented.  For now, LN commitment
```

Comment:
suggest s/never satisfactorily/not yet|not yet fully/ as per [comment](https://github.com/bitcoinops/bitcoinops.github.io/pull/394/files#r416206739)